src/Content/Widget/TrendingTags.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Friendica\Content\Widget;
+
+use Friendica\Core\Cache;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Database\DBA;
+use Friendica\Model\Term;
+
+class TrendingTags
+{
+	/**
+	 * @param string $content 'global' (all posts) or 'local' (this node's posts only)
+	 * @param int    $period  Period in hours to consider posts
+	 * @return string
+	 * @throws \Exception
+	 */
+	public static function getHTML($content = 'global', int $period = 24)
+	{
+		if ($content == 'local') {
+			$tags = self::getLocalTrendingTags($period);
+		} else {
+			$tags = self::getGlobalTrendingTags($period);
+		}
+
+		$tpl = Renderer::getMarkupTemplate('widget/trending_tags.tpl');
+		$o = Renderer::replaceMacros($tpl, [
+			'$title' => L10n::tt('Trending Tags (last %d hour)', 'Trending Tags (last %d hours)', $period),
+			'$more' => L10n::t('More Trending Tags'),
+			'$tags' => $tags,
+		]);
+
+		return $o;
+	}
+
+	/**
+	 * Returns a list of the most frequent global tags over the given period
+	 *
+	 * @param int $period Period in hours to consider posts
+	 * @return array
+	 * @throws \Exception
+	 */
```

Comment:
This seems for me like a `Friendica\Model\Tags` class method. I don't think we should mix it with Content methods.